lib/xeroizer/record/base.rb

```diff
@@ -105,12 +105,20 @@ def download_complete_record!
         end
 
         def save
-          return false unless valid?
+          save!
+          true
+        rescue XeroizerError
```

Comment:
Would be good to log here perhaps whatever XeroizerError messages are provided

Reply:
Good idea - I've added a log output within the rescue block. It's just using the existing `log` method so will be output as an info log rather than an error, but I can change this if this is a problem.